docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -186,6 +186,23 @@ Property | Description
 `node`  | The current DOM node.
 `idx` | Index of the current node among other nodes in the matching set.
 
+```js
+Selector('ul').filter((node, idx) => {
+    // node === a <ul> node
+    // idx === index of the current <ul> node
+}));
+```
+
+The `dependencies` parameter allows you to pass objects to the `filterFn` function's scope where they appear as variables.
```

Comment:
As a concept it will be pretty obvious. Just highlight that it's not an `option` object, but dependencies list itself